usr/bin/playtron-factory-reset

```diff
@@ -87,6 +87,9 @@ if [ -n "$PRESERVE_USER_FILES" ]; then
 else
 	# not preserving user files, delete home directory contents
 	rm -rf /home/playtron/{,.}*
+	if ! bootc status --format json | jq .status.booted.image.image.image | grep -P "^\"ghcr.io/playtron-os/playtron-os"; then
```

Comment:
Why is this conditioned on not preserving user files?

If we only want to do this on a full factory reset, we should probably change the name of the option for clarity.

But I feel like we should do this even on a partial reset.